driver/main.cpp

```diff
@@ -236,6 +237,18 @@ int main(int argc, char* argv[])
     {
         drv = new SumDriver<bfloat16, float>();
     }
+    else if(base_arg == "smoothl1loss")
```

Comment:
The driver format has changed. Please refer to https://github.com/ROCm/MIOpen/pull/2856 and modify it.